dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/validate-csv-data/rows/index.ts

```diff
@@ -0,0 +1,76 @@
+import { UtilisationReportCsvRowData, UtilisationReportDataValidationError, UtilisationReportFacilityData } from '@ukef/dtfs2-common';
+import addMatchingRowErrors from '../../helpers/add-matching-error-rows';
+
+/**
+ * validateRows
+ * Generate and returns validation errors for the facility utilisation csv rows
+ * if facilities have the same ukef facility ids, then the base currency and facility utilisation values should match
+ * if not, generate validation errors
```

Comment:
Is this "if the base currency and utilisation values do **not** match?

Reply:
Can we be more explicit in the documentation please, from reading this i'm not understanding if this is for 1 facility or multiple e.g should it be something like this?

(Purely an example and not representative of what it's actually doing): 
```
* Generate and returns validation errors for each facility in csv rows
* if multiple facilities have the same ukef facility ids,
* and any of those facilities "field A" and "field B" do NOT match
* then do XYZ for each facility
```

Reply:
changed